rosbag2_compression/test/rosbag2_compression/test_sequential_compression_writer.cpp

```diff
@@ -61,7 +61,7 @@ class SequentialCompressionWriterTest : public Test
 TEST_F(SequentialCompressionWriterTest, open_throws_on_bad_compression_format)
 {
   rosbag2_compression::CompressionOptions compression_options{
-    "bad_format", rosbag2_compression::CompressionMode::FILE};
+    "bad_format", rosbag2_compression::CompressionMode::FILE, 1, 4};
```

Comment:
nit: consider naming these (ex. `default_compression_queue_size`)

Reply:
Good point, I'll add consts for those.